lib/school_house_web/templates/page/index.html.heex

```diff
@@ -244,9 +244,9 @@
 <%= render(SchoolHouseWeb.PostView, "index.html", conn: @conn, posts: @posts) %>
 <div class="bg-body dark:bg-body-dark">
   <div class="px-4 py-16 mx-auto text-center max-w-7xl sm:px-6 lg:py-16 lg:px-8">
-    <h2 class="text-3xl font-extrabold tracking-tight text-heavy dark:text-heavy-dark sm:text-4xl">
-      <span class="block">Want to learn more?</span>
-    </h2>
+    <h1 class="text-3xl font-extrabold tracking-tight text-heavy dark:text-heavy-dark sm:text-4xl">
+      Want to learn more?
+    </h1>
```

Comment:
Then it could be `<h2>`.

Reply:
Yes, I think so, the first one "Welcome to..." should be `h1` and the rest `h2`.

Reply:
I changed `h2` to `h1` in here. e2fa6ad
`Welcome to part` is h1 in here, but `section` has to be changed to `div`  630a0b1.